scripts/lib/api/processHtml.ts

```diff
@@ -432,20 +432,11 @@ function processAttribute(
 
 function processFunctionOrException(
   $child: Cheerio<any>,
-  $dl: Cheerio<any>,
   id: string,
   githubSourceLink: string,
 ) {
-  const descriptionHtml = `<span class="target" id="${id}"/><p><code>${$child.html()}</code>${githubSourceLink}</p>`;
-
-  const pageHeading = $dl.siblings("h1").text();
-  if (id.endsWith(pageHeading) && pageHeading != "") {
-    // Page is already dedicated to apiType; no heading needed
-    return descriptionHtml;
-  }
-
-  const apiName = id.split(".").slice(-1)[0];
-  return `<h3>${apiName}</h3>${descriptionHtml}`;
```

Comment:
We still need the heading for now, and we always need the short name so that it shows up in the right page ToC, even after https://github.com/Qiskit/documentation/issues/948. 

Is this going to be computed in the docs app? Consider keeping the calculation here in this script. I think it makes sense to have the computation of all the relevant attributes live in this script at build-time, rather than dynamically by the app at runtime. That avoids wasted computation when the app is running, and keeps the docs app simpler that it doesn't need to know how to calculate a function name given its full import path.

It probably makes sense to set the short name via a prop and the docs app will for now create the h3 for you, if it's set. That might allow us to avoid having to regenerate the MDX files again once we implement https://github.com/Qiskit/documentation/issues/948. It'd be nice to only have to change the MDX this first time.